python/python/bystro/supervised_ppca/ppca_augmented.py

```
@@ -778,7 +794,198 @@ def get_covariance(self) -> NDArray[np.float_]:
+        Lambda : NDArray,(p,p)
+            The observational noise
+        """
+        if self.sigma2_ is None or self.p is None:
+            raise ValueError("Model has not been fit yet")
+
+        return self.sigma2_ * np.eye(self.p)
+
+    def _store_instance_variables(
+        self,
+        trainable_variables: tuple[
+            NDArray[np.float_], np.float_, 
```

Ruff

`np.float_` will be removed in NumPy 2.0. Use `numpy.float64` instead.

Replace with `numpy.float64`

(NPY201)